### PR TITLE
Fix for escaping in README

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ whether you want to follow PHP coding standards as defined in the PSR-1, PSR-2,
 or other community driven ones like the Symfony one.
 You can **also** define your (teams) style through configuration.
 
-It can modernize your code (like converting the `pow` function to the `**` operator on PHP 5.6)
+It can modernize your code (like converting the ``pow`` function to the ``**`` operator on PHP 5.6)
```
Comment (SpacePossum) [inline README.rst]: isn't this duplicating your PR here https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3408/files#diff-88b99bb28683bd5b7e3a204826ead112L941 ?

Comment (kubawerlos) [reply README.rst]: This was separate as it's not a fixer description. And the point of others was to remove `()` from function names. Here we could go with complex example i.e. `pow($foo, $bar)` and `$foo ** $bar`.

**Comment by SpacePossum** [comment]
@kubawerlos sorry for keep this one waiting, please rebase and we should be good to go I think

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**